routes/plants.js

```diff
@@ -1,49 +1,50 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const Plant = require('../models/Plant');
+const Plant = require("../models/Plant");
+const UserPlant = require("../models/UserPlant");
 
-router.get('/add', (req, res) => {
-  res.render('plants/add');
+router.get("/add", (req, res) => {
+  res.render("plants/add");
 });
 
-router.get('/', (req, res, next) => {
+router.get("/", (req, res, next) => {
   Plant.find()
-    .then(plants => {
-      res.render('plants/index', { plantsList: plants });
+    .then((plants) => {
```

Comment:
oh god this is why the plants index route wasn't working, YAS, thanks for catching it.